wren-modeling-rs/core/src/mdl/context.rs

```diff
@@ -19,6 +19,9 @@ use crate::logical_plan::utils::create_schema;
 use crate::mdl::manifest::Model;
 use crate::mdl::{AnalyzedWrenMDL, WrenMDL};
 
+/// Apply Wren Rules to the context for sql generation.
+/// TODO: There're some issue for unparsing the datafusion optimized plans.
+///   Disable all the optimize rule for sql generation temporarily.
```

Comment:
There's an issue about unparsing the optimized plans to the SQL string. I'll file the issue in Datafusion.